aspnetcore/fundamentals/configuration/index.md

```diff
@@ -217,7 +217,28 @@ setx Logging__1__Level=Information
 
 ### Environment variables set in launchSettings.json
 
-Environment variables set in *launchSettings.json* override those set in the system environment.
+Environment variables set in *launchSettings.json* override those set in the system environment. For example, the ASP.NET Core web templates generate a *launchSettings.json* file that sets the [Kestrel endpoint configuration](#kestrel) to:
```

Comment:
> that sets the [Kestrel endpoint configuration](#kestrel) to:

"applicationUrl" in launchSettings.json isn't Kestrel-specific. It should work with any server. The above makes it seem otherwise.

Furthermore, if there is any kestrel-specific endpoint configuration, it will override this (e.g. Kestrel endpoint config in appsettings.json overrides the "applicationUrl" configured in launchSettings.json). The section this links to describes this pretty well, but I could see the wording above confusing the reading if they don't follow the link.